app/viewmodels/CashAccountDailyStatementsViewModel.scala

```diff
@@ -42,15 +47,33 @@ case class CashAccountDailyStatementsViewModel(dailyStatements: Seq[DailyStateme
                                                noTransFromLastSixMonthsText: Option[HtmlFormat.Appendable] = None)
 
 object CashAccountDailyStatementsViewModel {
-  def apply(transactions: CashTransactions)(implicit msgs: Messages): CashAccountDailyStatementsViewModel = {
+  def apply(transactions: CashTransactions,
+            pageNo: Option[Int])(implicit msgs: Messages, config: AppConfig): CashAccountDailyStatementsViewModel = {
 
     val dailyStatements: Seq[CashDailyStatement] = transactions.cashDailyStatements.sortBy(_.date).reverse
     val hasTransactions = dailyStatements.nonEmpty
 
-    CashAccountDailyStatementsViewModel(populateDailyStatementViewModelList(dailyStatements),
+    CashAccountDailyStatementsViewModel(
+      dailyStatementsBasedOnPageNoForPagination(
+        populateDailyStatementViewModelList(dailyStatements), pageNo, config.numberOfRecordsPerPage
+      ),
       hasTransactions,
       transForLastSixMonthsHeading,
-      if(hasTransactions) None else Some(noTransFromLastSixMonthsText))
+      if (hasTransactions) None else Some(noTransFromLastSixMonthsText))
```

Comment:
there shouldn't be a space after the if before the (
should be: if(hasTransactions)

Reply:
Space after if is standard formatting, if(hasTransactions) is incorrect.

Reply:
I might be thinking of the html files, mb